### PR TITLE
Add some functions for hydrolysis support.

### PR DESCRIPTION
**Pull request by garlicnation**

### CODE REVIEW

File: dom5.js [hunk 1]
```diff
@@ -361,8 +394,8 @@ function append(parent, node) {
 }
 
 var parse5 = require('parse5');
-function parse(text) {
-  var parser = new parse5.Parser();
+function parse(text, options) {
+  var parser = new parse5.Parser(parse5.TreeAdapters.default, options);
```
Comment (azakus) [inline dom5.js]: Will the serializer also need `options` ?

**Comment by azakus** [comment]
LGTM with nit

File: dom5.js (hunk 1 above)
Comment (garlicnation) [inline dom5.js]: Maybe. I want to avoid it until I use it.

Comment (azakus) [inline dom5.js]: SGTM

**Pull request merged**